src/sendAPN.js

```diff
@@ -55,7 +55,7 @@ class APN {
       mdm: data.mdm,
       urlArgs: data.urlArgs,
       truncateAtWordEnd: data.truncateAtWordEnd,
-      collapseId: data.collapseKey,
+      collapseId: data.collapseId,
```

Comment:
breaking change to the lib's API, please revert

Reply:
same here